app/views/settings/_theme.haml

```diff
@@ -0,0 +1,135 @@
+  .card
+    .card-body
+      %h2 General
+      %p
+        Here you'll find general page values that are basically visible all across the page.
+      .row
+        .col-sm-6
+          = f.text_field :background_color, class: 'color', data: {default: 0xF0EDF4}
+        .col-sm-6
+          = f.text_field :body_text, class: 'color', data: {default: 0x000000}
+  .card
+    .card-body
+      %h2 Forms and Inputs
+      %p
+        Styles for form inputs, like textfields.
+      .row
+        .col-sm-6
+          = f.text_field :input_color, class: 'color', data: {default: 0xFFFFFF}
+        .col-sm-6
+          = f.text_field :input_text, class: 'color', data: {default: 0x000000}
+  .card
+    .card-body
+      %h2 Raised Content
+      %p
+        Raised content basically describes all the different boxes and panels you can see across the site.
+      .row
+        .col-sm-6
+          = f.text_field :raised_background, class: 'color', data: {default: 0xFFFFFF}
+        .col-sm-6
+          = f.text_field :raised_accent, class: 'color', data: {default: 0xF7F7F7}
+    .card-footer
+      %p Some text on top of a accented area on a raised element!
+  .card
+    .card-body
+      %h2 Colors
+      %p
+        Colors you can find used on the site. The names of the colors usually describe the context.
+        The "text" colors for all styles represent the color that text on top of these colors has.
+
+      %ul
+        %li
+          %b Primary:
+          The primary/brand color of the site, used for navigation, links, etc.
+        %li
+          %b Danger:
+          Color used for errors or critical actions like deleting something.
+        %li
+          %b Warning:
+          Color used for warnings if something non-critical has happened.
+        %li
+          %b Info:
+          Color used for informational popups or messages.
+        %li
+          %b Success:
+          Color used for messages if something went through successfully.
+      .row
+        .col-sm-6
+          = f.text_field :primary_color, class: 'color', data: {default: 0x5E35B1}
+        .col-sm-6
+          = f.text_field :primary_text, class: 'color', data: {default: 0xFFFFFF}
+        .col-sm-12
+          .alert.alert-primary
+            A simple primary alert—check it out!
+      .row
+        .col-sm-6
+          = f.text_field :danger_color, class: 'color', data: {default: 0xDC3545}
+        .col-sm-6
+          = f.text_field :danger_text, class: 'color', data: {default: 0xFFFFFF}
+        .col-sm-12
+          .alert.alert-danger
+            A simple danger alert—check it out!
+      .row
+        .col-sm-6
+          = f.text_field :warning_color, class: 'color', data: {default: 0xFFC107}
+        .col-sm-6
+          = f.text_field :warning_text, class: 'color', data: {default: 0xFFFFFF}
+        .col-sm-12
+          .alert.alert-warning
+            A simple warning alert—check it out!
```

Comment:
True, but that would require another migration for everyone running this already, wouldn't it, since the update-fields one is now a irreversible migration.

Reply:
Or does it?  We can keep the values users have set themselves, and change the default for users who have not changed their theme before or wanted a fresh start.

Reply:
Yeah, but changing the default in the database requires a migration.

...or an adjustment to the main one, which wouldn't apply it to people testing it right now.